lab_6/task_6_8.py

```diff
@@ -0,0 +1,16 @@
+import io
+import re
+
+
+def chapters_list():
+    with io.open('C:/Users/Andrii/Desktop/rob_crusoe.txt', 'r', encoding='utf-8') as r_crusoe,\
+         io.open('C:/Users/Andrii/Desktop/chapters.txt', 'w', encoding='utf-8') as list_chapters:
```

Comment:
in some cases you just used open,
in other you used one io.open and one without.
now you used 2 with io. please explain me  the principle how you chose?

Reply:
this is old version of code, on that moment I thought I need io for encoding, but encoding perectly work without io